rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2254,7 +2254,8 @@ def test_upgrade_db_41_to_42(user_data_dir, messages_aggregator):
         assert table_exists(cursor, 'calendar_reminders') is False
         assert cursor.execute('SELECT MAX(seq) FROM location').fetchone()[0] == 45
         assert cursor.execute('SELECT COUNT(tx_hash) FROM balancer_events').fetchone()[0] == 2
-        assert cursor.execute('SELECT COUNT(*) from used_query_ranges').fetchone()[0] == 2
+        assert cursor.execute('SELECT COUNT(*) FROM used_query_ranges').fetchone()[0] == 3
```

Comment:
1. What is the sqldiff of the test DB?
2. Can you make sure it has both yearn vaults and yearn vauls v2 events and used query ranges?
3. And can you do a more specific test here? That the used query rranges are there before, and that afterward they are not? Instead of just checking count?